app/controllers/merchant_items_controller.rb

```diff
@@ -34,9 +34,15 @@ def new
 
   def create
     merchant = Merchant.find(params[:merchant_id])
-    merchant.items.create(item_params)
-
-    redirect_to "/merchants/#{merchant.id}/items"
+    item = merchant.items.new(item_params)
+    if item.save
+      redirect_to "/merchants/#{merchant.id}/items"
+    else
+      message = item.errors.full_messages.to_sentence
+      flash[:notice] = "Item not created: #{message}"
+      
+      redirect_to "/merchants/#{merchant.id}/items"
```

Comment:
I'm not sure how it would work out best but I noticed that when I tried to submit an incomplete form that it still took me back to the merchant items index where is showed me the flash messages.  This could also stay on the form and display the message to give the user another attempt, though in our case it could cause the user to get stuck on the form if they changed their minds and needed to get back.

Reply:
We can do a redirect_to the page with the form, and it will display the flash message there. 
As far as the navigation/getting stuck, that won't be an issue if we put the nav bar from the wireframes on the page

Reply:
I had issues trying to do render :new so I skipped it.  I just tried it with redirect and that worked!